src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -3,6 +3,15 @@
 dataverse.version=${project.version}
 dataverse.build=
 
+# SEARCH INDEX
+dataverse.solr.host=localhost
+# Activating mp config profile -Dmp.config.profile=ct changes default to "solr" as DNS name
+%ct.dataverse.solr.host=solr
```

Comment:
What is this `%` for? Is it a typo? If not, can we please have a comment about out? Or is it documented in the guides already?

Reply:
Nope, this is not a typo, but activating using the value when profile "ct" is active (see your question/comment above)

https://download.eclipse.org/microprofile/microprofile-config-3.0/microprofile-config-spec-3.0.html#_how_config_profile_works

Reply:
Ok, makes more sense now. Thanks.